recipes/desman/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
```

Comment:
You might want to add `{% set version = "2.1" %}` to include `{{ version }}` below instead of hard-coded 2.1 to make it easier to update the recipe.

Reply:
Thanks @druvus for taking the time to review!